X/XSLT/build_tarballs.jl

```diff
@@ -34,7 +34,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Libgpg_error_jll", v"1.42.0"; compat="1.42.0"),
+    Dependency("Libgpg_error_jll"; compat="1.50"),
```

Comment:
Is there a newer version of xslt to build? We can't change an existing compat without changing the version number of the package itself (it's a Pkg limitation)

Reply:
Thankfully there is. Commit added to bump to v1.1.42.